certificate-policy.md

```diff
@@ -669,26 +656,22 @@ In the latter case, the OCSP signing Certificate shall contain an extension of t
 defined by RFC6960.
 
 ### 4.9.10 On-line revocation checking requirements
-The CA shall support an OCSP capability using the GET method for Certificates issued in accordance with these Requirements.
-
-For the status of Subscriber Certificates:
+The CA shall support an OCSP capability using the GET method for Certificates.
 
-- The OCSP Responder shall update the information used to respond to requests within 4 hours of a new CRL being issued by the CA for all requests. OCSP responses from this service shall have a maximum expiration time of ten (10) days
+For the status of Domain Validation TLS Server Authentication and Organization Validation TLS Server Authentication certificates, the CA shall update information provided via OCSP every 24 hours. OCSP responses shall have a maximum expiration time of seven (7) days.  
 
-For the status of Subordinate CA Certificates:
-
-- The CA shall update information provided via an Online Certificate Status Protocol at least (i) every 31 days and (ii) within 24 hours after revoking a Subordinate CA Certificate.
+For the status of Subordinate CA Certificates, the root CA shall update information provided via OCSP at least (i) every 31 days and (ii) within 24 hours after revoking a Subordinate CA Certificate.
 
 If the OCSP responder receives a request for status of a certificate that has not been issued, then the responder shall not respond with a "good" status. The CA shall monitor the responder for such requests as part of its security response procedures.
 
-OCSP responders for CAs which are not Technically Constrained in line with Section 7.1.5 shall not respond with a "good" status for such certificates.
-
 ### 4.9.11 Other forms of revocation advertisements available
-If the Subscriber Certificate is for a high-traffic FQDN, the CA may rely on stapling, in accordance with [RFC4366], to distribute its OCSP responses. In this case, the CA shall ensure that the Subscriber "staples" the OCSP response for the Certificate in its TLS handshake. The CA shall enforce this requirement on the Subscriber either contractually, through the Subscriber Agreement or Terms of Use, or by technical review measures implemented by the CA.
+If the Subscriber Certificate is for a high-traffic FQDN, the CA may rely on stapling, in accordance with RFC4366, to distribute its OCSP responses. In this case, the CA shall ensure that the Subscriber "staples" the OCSP response for the Certificate in its TLS handshake. The CA shall enforce this requirement on the Subscriber either contractually, through the Subscriber Agreement or Terms of Use, or by technical review measures implemented by the CA.
```

Comment:
Is this compatible with root store requirements, that you could exempt OCSP responses being available by `GET` requests if you can ensure stapling?

Reply:
@konklone The comment isn't a "nit"; your comment is valid.  

This policy statement is directly from the Baseline Requirements.  A related comment is in Issue #290 to "define a high-traffic FQDN".  I added your comments and mine to Issue #290.

Reply:
Thanks. And it was [my other comment](https://github.com/uspki/policies/pull/484#discussion_r158415034) that was the nit. :)